tests/package.json

```diff
@@ -17,6 +17,13 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "eslint": "^5.8.0",
```

Comment:
looks good, I just wonder why we aren't using the top-level eslint for this?

Reply:
Because the integration test image does not need to build all the rest of the services. 
I understand this is a bot confusing and we;ll probably need to reconsider how our docker works eventually.